decidim-core/config/locales/en.yml

```
@@ -1926,6 +1926,10 @@ en:
         mark_as_read: Mark as read
       offline_banner:
         cache_version_page: Oooops! Your network is offline. This is a previously cached version of the page you are visiting, perhaps the content is not up to date.
+      shared:
+        layout_item:
+          next: "next item"
+          prev: "previous item"
```

I have ran the normalization on my local, and the task suggests bot to have quotes. Also, i would consider these labels should be capitalized. 
```suggestion
          next: Next item
          prev: Previous item
```